lib/Echidna/Campaign.hs

```diff
@@ -121,27 +130,39 @@ evalSeq v e = go [] where
     case xs of []     -> pure ()
                (y:ys) -> e y >> go (y:r) ys
 
+-- | Execute a transaction, capturing the PC and codehash of each instruction executed, saving the
+-- transaction if it finds new coverage.
+execTxOptC :: (MonadState x m, Has Campaign x, Has VM x, MonadThrow m) => Tx -> m VMResult
+execTxOptC t = do
+  og  <- hasLens . coverage <<.= mempty
+  res <- execTxWith vmExcept (usingCoverage $ pointCoverage (hasLens . coverage)) t
+  hasLens . coverage %= unionWith union og
+  grew <- (== GT) . comparing coveragePoints og <$> use (hasLens . coverage)
```

Comment:
I did a few experiments with different contracts and I found that `grew` was `False` all the time. I'm investigating this..

Reply:
I believe the correct code should be: 
```
grew <- (== LT) . comparing coveragePoints og <$> use (hasLens . coverage)
```

Reply:
oops, good call